lib/src/models/submission_impl.dart

```diff
@@ -453,7 +477,6 @@ class SubmissionPreview {
   String _id;
 
   SubmissionPreview._fromMap(Map<String, dynamic> map) {
-    print('Preview: $map');
```

Comment:
Oops... thanks!

Reply:
Haha, I added this functionality really quickly and probably didn't write tests for it. Not sure how I missed it otherwise!